Source/AFError.swift

```diff
@@ -109,15 +109,15 @@ public enum AFError: Error {
     /// The underlying reason the response serialization error occurred.
     ///
     /// - inputDataNil:                    The server response contained no data.
-    /// - inputDataNilOrZeroLength:        The server response contained no data or the data was zero length.
+    /// - inputDataZeroLength:             The server response data was zero length.
     /// - inputFileNil:                    The file containing the server response did not exist.
     /// - inputFileReadFailed:             The file containing the server response could not be read.
     /// - stringSerializationFailed:       String serialization failed using the provided `String.Encoding`.
     /// - jsonSerializationFailed:         JSON serialization failed with an underlying system error.
     /// - propertyListSerializationFailed: Property list serialization failed with an underlying system error.
     public enum ResponseSerializationFailureReason {
         case inputDataNil
-        case inputDataNilOrZeroLength
+        case inputDataZeroLength
```

Comment:
That's my bad if you did. I'd say let's punt this to post 4.1.0. Maybe we can figure out a backwards compatible way to handle this feature before AF5.